src/algorithms/bubbleSort/BubbleSort.java

```diff
@@ -0,0 +1,22 @@
+package src.algorithms.bubbleSort;
+
+public class BubbleSort {
+    /**
+     * Sorts the given array.
```

Comment:
Perhaps can mention in ascending order too! Also, would be nice to give a brief description of how the algorithm works (eg. either talk about invariant), something like relies on 'bubbling' bigger elements to the right through neighbour-wise comparisons..

Talk about time complexity as well - feel free to use simple terms as well, like referencing outer and inner loops. Ultimately, its catered for 2040s students' learning

Reply:
Also, i realised this is in-place sorting. Do mention it if you're doing in-place!